cmd/mungedocs/util.go

```diff
@@ -141,6 +134,13 @@ type fileBlock struct {
 
 type fileBlocks []fileBlock
 
+var (
+	// Finds all preformatted block start/stops.
+	preformatRE    = regexp.MustCompile("^\\s*```")
+	notPreformatRE = regexp.MustCompile("^\\s*```.*```")
+	preformatEndRE = regexp.MustCompile("^\\s*```")
```

Comment:
We could distinguish that begin is ^```\s*.*\s*$ while end is ^```\s*$ ?

On Sat, Jul 18, 2015 at 11:57 AM, Daniel Smith <notifications@github.com>
wrote:

> In cmd/mungedocs/util.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11501#discussion_r34950388>
> :
>
> > @@ -141,6 +134,13 @@ type fileBlock struct {
> >
> >  type fileBlocks []fileBlock
> >
> > +var (
> > +	// Finds all preformatted block start/stops.
> > +	preformatRE    = regexp.MustCompile("^\\s*```")
> > +	notPreformatRE = regexp.MustCompile("^\\s*```.*```")
> > +	preformatEndRE = regexp.MustCompile("^\\s*```")
>
> Might as well just reuse preformatRE if they are the same. I had it
> separate to parse some (incorrect?) md files we had, but they've been
> fixed, I guess.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11501/files#r34950388>
> .
>